docs/whatsnew/1.0.0/1_0_0.rst

```diff
@@ -0,0 +1,81 @@
+Pylake 1.0.0
+============
+
+.. only:: html
+
+Pylake has made it to `v1.0.0`! And with it, several new features and improvements have been added.
+
+Documentation
+-------------
+
+.. image:: docs.png
+
+The documentation has been completely reworked.
+There is now a new :doc:`theory section</theory/index>` which delves into the theory of the underlying data analysis methods, including examples and references to literature for further reading.
+This information has been separated from the :doc:`tutorial section</tutorial/index>` so the tutorials can focus specifically on teaching you how to use Pylake.
+In addition, both sections were extended considerably.
+
+Every :doc:`tutorial</tutorial/index>` now contains links to example datasets hosted on `Zenodo <https://zenodo.org/>`_ that are downloaded directly when running the Jupyter notebooks.
+This allows you to immediately try out the various functionalities Pylake has to offer.
+
+Since `v0.13.0`, we have also reworked the `API documentation </api>` to have more information on the available functions and classes in Pylake.
+
+Creating kymographs from camera recordings
+------------------------------------------
+
+.. image:: kymo_to_stack.png
+
+A new feature introduced in `v1.0.0` is the ability to convert camera recordings to kymographs.
+
+First define a tether using :meth:`~lumicks.pylake.ImageStack.define_tether` which rotates the camera images such that the tether is aligned with the horizontal axis. Then, simply call :meth:`~lumicks.pylake.ImageStack.to_kymo`  to sum a region around the tether and construct a :class:`~lumicks.pylake.kymo.Kymo`.
```

Comment:
one line per sentence? also removed a spurious whitespace
~~~suggestion
First define a tether using :meth:`~lumicks.pylake.ImageStack.define_tether` which rotates the camera images such that the tether is aligned with the horizontal axis. Then, simply call :meth:`~lumicks.pylake.ImageStack.to_kymo` to sum a region around the tether and construct a :class:`~lumicks.pylake.kymo.Kymo`.
~~~